utils/wasm-builder/src/lib.rs

```diff
@@ -107,7 +107,7 @@ impl WasmBuilder {
         let profile = if profile == "debug" { "dev" } else { profile };
         self.cargo.set_profile(profile.to_string());
         self.cargo.set_features(&self.enabled_features()?);
-        if env::var("__GEAR_WASM_BUILDER_NO_PATH_REMAP").is_err() && profile != "dev" {
+        if env::var("__GEAR_WASM_BUILDER_NO_PATH_REMAP").is_err() && profile == "production" {
```

Comment:
This more resembles exlicit feature and has less flexibility then a dedicated feature. If it is the only option we can suggest to our customers, I would suggest introducing the latter, so they can decide themselves whether and when they want the paths to be remapped.